src/components/shared/MarkerSettings.css

```diff
@@ -13,20 +13,30 @@
   white-space: nowrap;
 }
 
-.filterMarkersButton {
+.filterMarkersButton,
+.copyTableButton {
   position: relative;
   width: 24px;
   height: 24px;
   flex: none;
   padding-right: 30px;
   margin: 0 4px;
-  background-image: url(../../../res/img/svg/filter.svg);
   background-position: 4px center;
   background-repeat: no-repeat;
 }
 
+.filterMarkersButton {
+  background-image: url(../../../res/img/svg/filter.svg);
+}
+
+.copyTableButton {
+  margin: 0 16px 0 4px;
```

Comment:
nit: `margin-right: 16px` should be enough since we handle the other margins in the block above.